src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
What is the state of sequence annotation methods?  CH, if you
have everything you are working on pushed, I can take pass
today.

I would love to put in a pull request ASAP.

Mark